inginious-problems-blockly/static/task_blockly.js

```diff
@@ -26,4 +26,14 @@ function load_input_blockly(submissionid, key, input) {
         blocklyxmlarea.prop('value', input["blockly"]);
     }
 
-}
\ No newline at end of file
+}
+
+function load_feedback_blockly(key, content){
+
+    if (typeof Maze !== "undefined" && typeof Maze.reset !== "undefined" && content[0] !== "success") {    
+        var test = $('<div>');
```

Comment:
We should mention in comments here that this is a temporary workaround ;-)

Reply:
I done it ;)

Reply:
Just! I done it now ;)